spec/components/previews/provider_interface/application_course_summary_component_preview.rb

```diff
@@ -0,0 +1,10 @@
+module ProviderInterface
+  class ApplicationCourseSummaryComponentPreview < ViewComponent::Preview
+    layout 'previews/provider'
+
+    def course_details
+      application_choice = build_stubbed(:application_choice)
+      render ApplicationCourseSummaryComponent.new(application_choice:)
```

Comment:
I get an error when I try to view this at `view_components/provider_interface/application_course_summary_component/course_details`

<img width="1395" alt="image" src="https://github.com/user-attachments/assets/c5b99c74-eded-4a18-8b9f-509795cc361d">